lib/Parse/ParseExpr.cpp

```diff
@@ -3297,8 +3293,18 @@ ExprResult Parser::ParseGenericTypeArgumentList(ExprResult Res, SourceLocation L
   }
   ConsumeToken(); // consume '>' token
 
-  auto TypeArgs = ArrayRef<DeclRefExpr::GenericInstInfo::TypeArgument>(typeArgumentInfos);
-  return Actions.ActOnTypeApplication(Res, Loc, TypeArgs);
+  return std::make_pair<>(false, typeArgumentInfos);
```

Comment:
I think you can omit the <> in std::make_pair?

Reply:
Yep. Done.